corehq/apps/hqwebapp/templates/home.html

```diff
@@ -10,8 +10,8 @@
 {% endblock %}
 {% block head %}{{ block.super }}
     <link rel="stylesheet" href="{% static 'hqwebapp/css/mainlogin.css' %}"/>
-    <link rel="stylesheet" href="{% static 'hqwebapp/stylesheets/blueprint/screen.css' %}" media="screen, projection">
```

Comment:
yeah I think we can get rid of it. there was a time when prelogin was a static site you had to build separately but now that it's shipped with HQ i think we can delete this.